qa/workunits/ceph-helpers.sh

```diff
@@ -1023,32 +1033,36 @@ function test_is_clean() {
 
 ##
 # Wait until the cluster becomes clean or if it does not make progress
-# for $TIMEOUT seconds. The function **is_clean** is used to determine
-# if the cluster is clean. Progress is measured either vian the
-# **get_is_making_recovery_progress** predicate or if the number of
-# clean PGs changes.
+# for $TIMEOUT seconds.
+# Comparing get_num_active_clean & get_num_pgs is used to determine
+# if the cluster is clean. That's almost an inline of is_clean() to
```

Comment:
the documentation is designed to be read by the user and should not contain implementation details. it should focus on documenting precisely each aspect of function behavior and error handling, pitfall etc. 

the reason why is_clean is inlined could usefuly be added as a comment within the function itself, as a reminder for the implementer